cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORGenerator.java

```diff
@@ -402,9 +417,20 @@ public final void writeStringField(String fieldName, String value)
             return;
         }
         _verifyValueWrite("write String value");
+        _decrementList();
         _writeString(value);
     }
 
+	
+    public final void writeFieldLong(long size) throws IOException {
```

Comment:
We are implementing a protocol that use number as key in Maps. 
For more information: http://core-wg.github.io/yang-cbor/draft-veillette-core-cool-latest.html 

I understand that this method won't be exposed via the streaming API but in our case, we are currently using the CBORGenerator directly (we are not using the ObjectMapper). 

Do you think it could be feasible to modify the jackson-databind project to provide such flexibility? Using annotations for example.